.codeclimate.yml

```diff
@@ -34,6 +34,12 @@ engines:
   rubocop:
     enabled: true
     config: '.rubocop_cc.yml'
+prepare:
+  fetch:
+  - url: "https://raw.githubusercontent.com/ManageIQ/manageiq/master/.rubocop_base.yml"
+    path: ".rubocop_base.yml"
+  - url: "https://raw.githubusercontent.com/ManageIQ/manageiq/master/.rubocop_cc.yml"
+    path: ".rubocop_cc.yml"
```

Comment:
in other words, this PR drops all changes on top of the main repos versions... how do we apply any customizations we want for a specific repo?